Dockerfile

```diff
@@ -1,25 +1,18 @@
 FROM docker.io/library/python:3.11
 
-# Copy firewatch package into /firewatch folder
-RUN mkdir /firewatch
-COPY . /firewatch/
+COPY pyproject.toml poetry.lock README.md /firewatch/
+COPY cli /firewatch/cli/
 
-# Install firewatch
-RUN pip install --upgrade pip
-RUN pip install /firewatch tox
+WORKDIR /firewatch
 
-# Copy the Jira config template to /templates
-RUN mkdir /templates
-COPY cli/templates/* /templates/
+ENV POETRY_HOME=/firewatch
+ENV PATH="/firewatch/bin:$PATH"
 
-# Add permissions
-RUN chgrp -R 0 /tmp && \
-    chmod -R g=u /tmp
+RUN python3 -m pip install pip poetry --upgrade \
+    && poetry config cache-dir /openshift-cli-installer \
+    && poetry config virtualenvs.in-project true \
+    && poetry config installer.max-workers 10 \
+    && poetry install
 
-RUN chgrp -R 0 /firewatch && \
-    chmod -R g=u /firewatch
 
-RUN chgrp -R 0 /templates && \
-    chmod -R g=u /templates
-
-CMD ["/bin/bash"]
+ENTRYPOINT ["poetry", "run", "firewatch"]
```

Comment:
```suggestion
# Add permissions
RUN chgrp -R 0 /tmp && \
    chmod -R g=u /tmp && \
    chgrp -R 0 /firewatch && \
    chmod -R g=u /firewatch
    
ENTRYPOINT ["firewatch"]
```
I believe we need to add permissions for execution inside of OpenShift. Also changing the `ENTRYPOINT` to reflect the suggested change above.

Reply:
Are you sure this is needed? do you write to anywhere beside /tmp? 
We have a few containers running on openshift CI and never had to do it.

Reply:
@myakove I'll give it a test in a cluster this morning and get back to you

Reply:
@myakove you are correct this is not needed in the firewatch Dockerfile :)